RTC beta (#1778)
tinymce/tinymce-docs: tinydrive/libraries/dotnet.md

#### 2. Change the `apiKey` in `tinydrive-dotnet-mvc-starter/appsettings.Development.json`

> You get the API key from the `API Key Manager` {{site.accountpage}} page at http://tiny.cloud.

Reviewer: ```suggestion
The API key can be found in the `API Key Manager` {{site.accountpage}} page at http://tiny.cloud.
```
maybe?
